shell/platform/fuchsia/flutter/tests/integration/embedder/flutter-embedder-test.cc

```diff
@@ -66,7 +64,8 @@ constexpr char kChildViewUrl[] =
 constexpr char kParentViewUrl[] =
     "fuchsia-pkg://fuchsia.com/parent-view#meta/parent-view.cm";
 static constexpr auto kTestUiStackUrl =
-    "fuchsia-pkg://fuchsia.com/test-ui-stack#meta/test-ui-stack.cm";
+    "fuchsia-pkg://fuchsia.com/flatland-scene-manager-test-ui-stack#meta/"
```

Comment:
It's important to avoid encoding these implementation details in the name.  This is in fact why alex chose to preserve the "test-ui-stack" name.

You can update the "test-ui-stack" package definition here to include the flatland-scene-manager config: https://source.corp.google.com/h/turquoise-internal/turquoise/+/main:src/ui/testing/test_ui_stack/BUILD.gn;l=82?q=test_ui_stack%2FBUILD.gn

Ideally, we could do that before landing this CL.

If that's not possible and we must wait until after this CL (due to soft transitions), then I insist that we file a bug and follow-up with another PR here to change the name to "test-ui-stack".

Reply:
Actually, we are going to have to come back here anyways with a followup, so it's no big deal to do this now.

Test UI Stack will be replaced with the UI Test Realm Proxy, which will abstract these details.  Additionally, we will use subpackages to run the "touch-input-test" from in-tree here and eliminate the duplicated code.

The naming issues can be fixed at that time, so it's OK to defer them.

Reply:
Nice, thanks for the heads up!